frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,29 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-cd-title {
+    margin-top: var(--pf-global--spacer--md);
+  }
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-empty-state {
+    flex-direction: column;
+    align-items: start;
+    margin-top: var(--pf-global--spacer--sm);
+    padding-left: var(--pf-global--spacer--md);
+
+    .pf-c-title {
+      color: var(--pf-c-wizard__nav-link--m-disabled--Color);
```

Comment:
shouldn't we rather use console's `text-secondary` here? @matthewcarleton

Reply:
yup, I'd suggest `--pf-global--Color--200` here

Reply:
I think we should reference pf classes only when you cannot pass a kubevirt specific class to some child. Meaning only when you are forced to use pf one